napalm_eos/eos.py

```diff
@@ -985,16 +985,20 @@ def get_mac_address_table(self):
     def get_route_to(self, destination='', protocol=''):
         routes = dict()
 
+        if protocol.lower() not in ['', 'bgp', 'connected', 'isis', 'ospf', 'rip', 'static']:
```

Comment:
More than happy to, was just trying to match up closer to the junos reference spec.  Do we want to try to throw a special UnknownProtocol exception or let the command parser raise an exception when an unknown protocol is passed in, instead?